salt/metalk8s/roles/etcd/prepared.sls

```diff
@@ -0,0 +1,4 @@
+include:
+  - metalk8s.roles.internal.node-without-calico
+  - metalk8s.container-engine.containerd
```

Comment:
Hmmm it's already part of `metalk8s.roles.internal.node-without-calico` so not needed